inc/saiacl.h

```diff
@@ -1154,7 +1154,8 @@ typedef struct _sai_acl_api_t
     sai_remove_acl_range_fn             remove_acl_range;
     sai_set_acl_range_attribute_fn      set_acl_range_attribute;
     sai_get_acl_range_attribute_fn      get_acl_range_attribute;
-
+    sai_create_acl_group_fn             create_acl_group;
```

Comment:
@tushar-ty - I have updated the spec/document with review comments as well. Please find the updated signature in the PR, that lists the following signatures for the two functions: 

typedef sai_status_t (*sai_create_acl_table_group_fn)(                              
        _Out_ sai_object_id_t *acl_table_group_id,                                  
        _In_ uint32_t attr_count,                                                   
        _In_ const sai_attribute_t *attr_list);                                     

typedef sai_status_t (*sai_remove_acl_table_group_fn)(                              
        _In_ sai_object_id_t acl_table_group_id);                                   


The attributes used for sai_create_acl_table_group_fn : SHOULD be updated ENUM defined by : sai_acl_table_group_attr_t.